code/modules/clothing/under/miscellaneous.dm

```diff
@@ -29,6 +29,18 @@
 		to_chat(user, "<span class='warning'> You aren't sure you'll fit in this men's cloth..</span>")
 		return 0
 
+/obj/item/clothing/under/german
+	name = "ancient uniform"
+	desc = "A uniform of an ancient officer. How did it survived to there days?"
```

Comment:
An uniform. Ну и сэйм щит как сверху

Reply:
> An uniform. Ну и сэйм щит как сверху

The word uniform begins with a palatal approximant /j/. The palatal approximant is a consonantal sound, even though the letter itself is a vowel. Since we use the pronunciation of the word following the article to determine whether we use "a" or "an", and because it is pronounced starting with a consonant, we use "a". Thus, the original sentence is correct.

сори ваш аргумент инвалид

Reply:
окей, инвалид так инвалид